kernel/build/build_deb

```diff
@@ -88,11 +88,17 @@ echo "*****"
 # Clean the git repo
 git clean -fxd
 
+# Install build dependencies
+sudo apt-get install -y bc
+
 # Prep the config
 echo "Preparing the kernel config"
-cp "$(dirname $0)/kernel-config-deb.basic" .config
+wget https://raw.githubusercontent.com/ceph/ceph-build/master/kernel/build/kernel-config-deb.basic -O .config
```

Comment:
Actually, I was hoping I could use this plugin

https://wiki.jenkins-ci.org/display/JENKINS/Config+File+Provider+Plugin

which seems to be designed to cover exactly this kind of use case but I'm not familiar with the yaml syntax for it and it does not seem to be enabled in our Jenkins instance at the moment. I've seen the plugin being available in the UI but I did not change anything there since I didn't want to break anything -- I'm not sure about what we use underneath the Jenkins to generate these things.  Cloning the entire ceph-build repo just for some config files seems like a bit of an overkill to me.